### ISSUE TITLE
Add abiilty to load JDK cacerts file

### ISSUE THREAD

**Issue by maxxedev** [Hakky54/sslcontext-kickstart#502]
**Is your feature request related to a problem? Please describe.**
Add ability to easily load JDK cacerts file as a KeyStore. As is now, it could be loaded like this:
```java
X509ExtendedTrustManager trustManager = TrustManagerUtils.createTrustManagerWithJdkTrustedCertificates();
KeyStoreUtils.createTrustStore(trustManager.getAcceptedIssuers());
```
... although cacerts, like any KeyStore, can technically contain non-Certificates.

**Describe the solution you'd like**
Define a `KeyStoreUtils.loadJdkCacertsKeyStore()` or similar.

Ideally, replicate this code:
https://github.com/openjdk/jdk/blob/9a83dfee14f4cd9cda476d11a027294a810953cb/src/java.base/share/classes/sun/security/ssl/TrustStoreManager.java#L63-L65
https://github.com/openjdk/jdk/blob/9a83dfee14f4cd9cda476d11a027294a810953cb/src/java.base/share/classes/sun/security/ssl/TrustStoreManager.java#L132-L141

based on documentation:
https://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/JSSERefGuide.html#CustomizingStores

Or, re-use `createTrustManagerWithJdkTrustedCertificates()`

**Comment by Hakky54** [comment]
@maxxedev Thank you for requesting to provide the jdk keystore within the KeyStoreUtils. It seems like you want to just get a KeyStore instance with the jdk trusted certificates just from the KeyStoreUtils. I created it, see here for the PR: https://github.com/Hakky54/sslcontext-kickstart/pull/503

Just out of curiousity, why would you need a separate KeyStore instance, what is your use case? Currently I am providing the jdk trusted certificate from the SSLFactory merged in a SSLContext, CertificateUtils as a list of trusted certtificates and from the TrustManagerUtils as a TrustManager, It seems like your use case is slightly different and I am curious how you will use this KeyStore instance.

**Comment by maxxedev** [comment]
Thanks for a quick PR

In some cases, TrustManagerFactory needs to be initilized with [`init(ManagerFactoryParameters spec)`](https://docs.oracle.com/javase/8/docs/api/javax/net/ssl/ManagerFactoryParameters.html). And it's not possible to pass null KeyStore to [PKIXParameters](https://github.com/openjdk/jdk/blob/bc7d9e3d0bc663bbbeb068889082da4a9f0fa8de/src/java.base/share/classes/java/security/cert/PKIXParameters.java#L146-L148).

[ManagerFactoryParameters](https://docs.oracle.com/javase/8/docs/api/javax/net/ssl/ManagerFactoryParameters.html) -> [CertPathTrustManagerParameters](https://docs.oracle.com/javase/8/docs/api/javax/net/ssl/CertPathTrustManagerParameters.html) -> [CertPathParameters](https://docs.oracle.com/javase/8/docs/api/java/security/cert/CertPathParameters.html) -> [PKIXParameters](https://docs.oracle.com/javase/8/docs/api/java/security/cert/PKIXParameters.html)

**Comment by Hakky54** [comment]
Maybe I am missing something. So is the code change in the PR ok for you?

> In some cases, TrustManagerFactory needs to be initilized with init(ManagerFactoryParameters spec). And it's not possible to pass null KeyStore to PKIXParameters.

So are you suggesting that the implementation should be something else?

**Comment by maxxedev** [comment]
PR is very ok. I was answering your question:
> Just out of curiousity, why would you need a separate KeyStore instance, what is your use case?

**Comment by Hakky54** [comment]
This is now available in version 8.3.6
